### PR TITLE
Previous Experiments

### PR DESCRIPTION
**Pull request by amirrr**
This will add an experiment that contains the best highly rated statements from 67 design points from previously ran experiment in the original paper.

TODO: Add logo to footer

### CODE REVIEW

File: server/controllers/experiment.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ const returnStatements = async (req, res) => {
     )
     .filter((treatment) => treatment.validity({ ...req }));
 
+  console.log(oursobject);
```
Comment (markwhiting) [inline server/controllers/experiment.js:28]: Probably can drop, right?

File: server/survey/experiments/previous.experiment.js [hunk 2]
```diff
@@ -0,0 +1,1220 @@
...
+      ids: [
+        3453, 1659, 2680, 1123, 1552, 2889, 1805, 2346, 541, 1534, 3772, 179,
+        254, 1848, 1727,
+      ],
+    },
+    function: GetStatementById,
+  },
+];
+
+const experiment = {
```
Comment (markwhiting) [inline server/survey/experiments/previous.experiment.js:1212]: Shouldn't there be some kind of assignment handling here? e.g., so turk people can see this experiment and others can not?

File: server/survey/experiments/previous.experiment.js [hunk 3]
```diff
@@ -0,0 +1,1220 @@
...
+  experimentName: "design-design_point-old-statements",
+  treatments: defaultTreatment,
+  treatmentAssigner: (subject, treatments) => {
+    return treatments[Math.floor(Math.random() * treatments.length)];
```
Comment (markwhiting) [inline server/survey/experiments/previous.experiment.js:1216]: This is unweighted random, right? (which is fine, just wondering if this is the place that the assignment is occurring that we were talking about yesterday)

**Review by markwhiting — CHANGES_REQUESTED** [review]
A few minor comments. 

Also, think we can do the logos before launching this one?

File: server/survey/experiments/previous.experiment.js (hunk 3 above)
Comment (amirrr) [reply server/survey/experiments/previous.experiment.js:1216]: you are right, the treatment assigner is not being used from here. We use the one from controllers/experiment.js

File: server/survey/experiments/previous.experiment.js (hunk 2 above)
Comment (amirrr) [reply server/survey/experiments/previous.experiment.js:1413]: Thanks for noticing. fixed now.

**Comment by amirrr** [comment]
footer light mode:

image

footer dark mode:
image

**Review by markwhiting — APPROVED** [review]

**Pull request merged**